jib-maven-plugin/src/main/java/com/google/cloud/tools/jib/maven/skaffold/FilesMojoV2.java

```diff
@@ -167,34 +167,53 @@ public void execute() throws MojoExecutionException, MojoFailureException {
     }
   }
 
-  private List<Path> resolveExtraDirectories(MavenProject project) {
+  private List<Path> resolveExtraDirectories(MavenProject project) throws MojoExecutionException {
     // Try getting extra directory from project/session properties
-    String extraDirectoryProperty =
+    String deprecatedProperty =
         MavenProjectProperties.getProperty(PropertyNames.EXTRA_DIRECTORY_PATH, project, session);
-    if (extraDirectoryProperty != null) {
-      return Collections.singletonList(Paths.get(extraDirectoryProperty));
+    String newProperty =
+        MavenProjectProperties.getProperty(PropertyNames.EXTRA_DIRECTORIES_PATHS, project, session);
+
+    if (deprecatedProperty != null && newProperty != null) {
+      throw new MojoExecutionException(
+          "You cannot configure both 'jib.extraDirectory.path' and 'jib.extraDirectories.paths'");
+    }
+
+    String property = newProperty != null ? newProperty : deprecatedProperty;
+    if (property != null) {
+      return Collections.singletonList(Paths.get(property));
```

Comment:
Hmm... looks like I missed this part. The new property can be a list.

Reply:
So I missed handling this as list in the previous PR at this location: https://github.com/GoogleContainerTools/jib/pull/1602/files#diff-1aee6421b98ba15074ed3f0da4755bc5R175

Fixed with the latest commit.